core/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommit.java

```diff
@@ -144,15 +141,13 @@ public void commit() throws CommitException, UnknownTransactionStatusException {
               + " before calling commit()");
     }
 
-    if (isCoordinator) {
-      try {
-        commit.commitState(crud.getSnapshot());
-      } catch (Exception e) {
-        // no need to rollback because the transaction has already been rolled back
-        needRollback = false;
+    try {
+      commit.commitState(crud.getSnapshot());
```

Comment:
Regardless that it's a coordinator process or a participant process, we always call this `commit.commitState()` so that we can call it in parallel in all coordinator/participant processes.

Only one process can succeed to putting a transaction state to a coordinator table as follows:
https://github.com/scalar-labs/scalardb/blob/allow-commit-and-rollback-to-be-called-in-parallel-in-2pc/core/src/main/java/com/scalar/db/transaction/consensuscommit/CommitHandler.java#L114

And the rest of the processes catch `CoordinatorConflictException`, and they need to get a transaction state:
https://github.com/scalar-labs/scalardb/blob/allow-commit-and-rollback-to-be-called-in-parallel-in-2pc/core/src/main/java/com/scalar/db/transaction/consensuscommit/CommitHandler.java#L119

So, this change can impact the performance a bit.